shared/ui-components/src/forms/Form.tsx

```diff
@@ -13,7 +13,7 @@ const Form = ({ id, children, className, onSubmit }: FormProps) => {
   }
 
   return (
-    <form id={id} className={className} onSubmit={onSubmit} onKeyPress={onKeyPress}>
+    <form id={id} className={className} onSubmit={onSubmit} onKeyPress={onKeyPress} noValidate>
```

Comment:
I think I know why this is here, but I want to make sure this gets vetted by @slowbot.

Reply:
Yea, it's definitely a thing we need to re-test whole the entire form. Not sure where, but @slowbot added info about this, and it's reasonable for me - e.g. custom phone validation doesn't work when noValidate attr is missing.